### DISCUSSION TITLE
solarmanpv.com

### DISCUSSION THREAD

**Discussion by dabrich** [hultenvp/solis-sensor#319] [Q&A]
My integration was working with "home.solarman.cn". Now it was migrated to "home.solarmanpv.com" and it stopped working. What should I do?

**Comment by hultenvp** [comment]
See #317

**Reply by dabrich** [reply]
That means that solis-sensor won't work anymore, right?

**Reply by hultenvp** [reply]
With the limited investigation I've done I'd say that's indeed the case.

**Reply by dabrich** [reply]
since it was working before, is somebody working to get this functionality back with the new server?

**Reply by hultenvp** [reply]
I do not know if someone is creating a new integration for the solarmanPV cloud.
The reason the solis solution worked was because the old solis service and the old solarman service were essentially the same applications with different branding.
That seems no longer the case for the new services.
Have a look here as there is some community activity: https://community.home-assistant.io/t/collect-inverter-data-from-solarman-api/328760

**Reply by dabrich** [reply]
I've checked that and unfortunately I'm not able to get all data. Anyway, thank you and now it's clear that it won't be supported here.
